### DISCUSSION TITLE
What is ANDROID app?

### DISCUSSION THREAD

**Discussion by MicroBeast** [celzero/rethink-app#1751] [Q&A] [answered]
![Screenshot_20241026_151108](https://github.com/user-attachments/assets/b1abd336-5104-4936-b505-93876b481e3f)

There is in Rethink list an app (or something) that called ANDROID (all capital). It eventually will contact alot of places outside OEM URLs. Other apps like InviZible or NekoBox doesn't listed that. Or probably I don't know its equivalent name. It's not "Android System".

I forgot what was that in Rethink. I roughly remember (that if I'm not mistaken) it ever talked on Telegram months ago. Searching for word "ANDROID" returns too many results.

Thanks.

**Comment by ignoramous** [comment, marked as answer]
> Component that shows up as `ANDROID` is *not* spyware. It is your OS.
>
> It does send some network requests via the tunnel Rethink sets up. It isn't an app (in that, it isn't "installed"), but like apps it does have a UID (`0`) and other attributes.
>
> Other similar OS components you may come across, `MDNSR`, `GPS`, `DNS` etc. There are some 100+ of them: https://android.googlesource.com/platform/system/core/+/master/libcutils/include/private/android_filesystem_config.h

From: https://www.reddit.com/r/rethinkdns/comments/1bd31ec/an_unknown_app_installed_by_itself_called_0_or/ / https://archive.md/4AcEl
